packages/@magic-sdk/react-native-bare/test/mocks.ts

```diff
@@ -6,10 +44,9 @@ export function reactNativeStyleSheetStub() {
 const noopModule = () => ({});
 
 export function removeReactDependencies() {
-  jest.mock('react', noopModule);
```

Comment:
Is there a reason you chose to remove this line? In other words, is there a reason we'd previously removed the react mock on setup?

Reply:
I'm not sure why it was mocked, normally in RN projects you don't mock all of React. Also, removing this line doesn't cause any issues with the tests. Leaving it as it was does cause issues when testing things like hooks that come from React (useState, useEffect, etc.).